### PR TITLE
Add video link to net MVC sample

### PR DESCRIPTION
**Pull request by Jay-umbr**
Added a link to https://www.youtube.com/watch?v=yTB-tXi3Ldg under https://our.umbraco.com/documentation/Umbraco-Heartcore/Client-Libraries/Dot-Net-Core/MVC-Sample/

### CODE REVIEW

File: Reference/Security/Security-hardening/index.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ Some older versions of Umbraco also relied on /umbraco/webservices/ for loadbala
 
 If your server is behind a load balancer, you should use `{HTTP_X_FORWARDED_FOR}` instead of `{REMOTE_ADDR}` as the input for the rule.
 
+In some server configurations you will need to allow the server that is running the Umbraco backoffice to be able to make internal requests to the /umbraco folder. To do that, you will need to add the servers IP address to the rewrite rule as negations.
```
Comment (sofietoft) [inline Reference/Security/Security-hardening/index.md:51]: Looks like something that was added by mistake? 🤔

**Review by sofietoft — COMMENTED** [review]
Thanks for adding the link @Jay-umbr !

Looks like something for another article made it's way into this PR.
Could you double-check that?

File: Reference/Security/Security-hardening/index.md (hunk 1 above)
Comment (Jay-umbr) [reply Reference/Security/Security-hardening/index.md:51]: uhh I do not know how that got there- I did not write that

Comment (Jay-umbr) [reply Reference/Security/Security-hardening/index.md:51]: can I just delete that file and all would be good?

Comment (sofietoft) [reply Reference/Security/Security-hardening/index.md:51]: Strange. 
You should remove that part of the file, not the entire file.
Removing the entire file, will also remove it from the docs 🙈

**Comment by Jay-umbr** [comment]
@sofietoft should be all good now

**Comment by sofietoft** [comment]
Wonderful @Jay-umbr !

I'll get it merged right away.

**Pull request merged**